index.js

```diff
@@ -1,13 +1,17 @@
 //@noflow
 const styles = require('./styles');
-const React = require('react-native');
-const { Dimensions, Animated, } = React;
+const ReactNative = require('react-native');
+const React = require('react');
+const { Dimensions, Animated, } = ReactNative;
 const deviceScreen = Dimensions.get('window');
 
 const {
   PanResponder,
   View,
   TouchableWithoutFeedback,
+} = ReactNative;
+
+const {
   Component,
 } = React;
```

Comment:
Can we replace `extends Component` with `React.Component` ? I am pretty sure it's not that needed to destruct it here, @Kureev ?